dll/cpl/mmsys/lang/pt-PT.rc

```diff
@@ -162,16 +170,16 @@ END
 
 IDD_SETUP1 DIALOGEX 0, 0, 227, 206
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
-CAPTION "Configuração"
+CAPTION "Definições"
```

Comment:
What's the difference between these two terms (the old and new ones)?

Reply:
"Configurações" is a more generic term.
 Windows most commonly uses the term "Definições".